crates/tests-e2e/tests/receive_payment_when_open_position.rs

```diff
@@ -0,0 +1,26 @@
+use native::api;
+use tests_e2e::setup;
+use tests_e2e::wait_until;
+use tokio::task::spawn_blocking;
+
+#[tokio::test]
+#[ignore = "need to be run with 'just e2e' command"]
+async fn can_receive_payment_with_open_position() {
+    let test = setup::TestSetup::new_with_open_position().await;
+    let app = &test.app;
+
+    let ln_balance_before = app.rx.wallet_info().unwrap().balances.lightning;
+    let invoice_amount = 10_000;
+
+    tracing::info!("Creating an invoice");
+    let invoice = spawn_blocking(move || api::create_invoice_with_amount(invoice_amount))
+        .await
+        .unwrap()
+        .unwrap();
+
+    tracing::info!("Coordinator pays the invoice created in the app");
+    test.coordinator.pay_invoice(&invoice).await.unwrap();
+
+    wait_until!(app.rx.wallet_info().unwrap().balances.lightning > ln_balance_before);
```

Comment:
❓ is there a reason to why you are using `>` rather than `==`

Reply:
it's not an assert, it's wait. it's enough to notice the balance change before the actual assertion. 

I don't mind changing it, if this is stops you from approving it.